Corgibytes.Freshli.Lib/Util/DateTimeExtensions.cs

```diff
@@ -2,8 +2,30 @@
 
 namespace Corgibytes.Freshli.Lib.Util {
   public static class DateTimeExtensions {
```

Comment:
Should this be changed to `DateTimeOffsetExtensions`?

Reply:
Good point. I'll fix that.